serverless/functions/join-stream-as-speaker.js

```diff
@@ -72,25 +72,44 @@ module.exports.handler = async (context, event, callback) => {
   }
 
   const streamSyncClient = client.sync.services(streamMapItem.data.sync_service_sid);
-  
+
+  const viewerDocumentName = `viewer-${user_identity}`;
+  // Create viewer document
   try {
-    // Reset the viewers hand_raised and speaker_invite status
-    await streamSyncClient.documents(`viewer-${user_identity}`).update({ data: { speaker_invite: false } });
```

Comment:
We were updating the viewer document to clear the speaker invite but I don't think we needed to. This should only be important when the user is a viewer and that happens in `join-stream-as-viewer`.